### ISSUE TITLE
chore: add resuable Navbar component to frontend cert (react workshop)

### ISSUE THREAD

**Issue by jdwilkin4** [freeCodeCamp/CurriculumExpansion#543]
Here are the changes that Oliver has been working on for the react projects in the fcc editor.
https://github.com/freeCodeCamp/freeCodeCamp/pull/55860

For this first React workshop, all JSX will need to go in the HTML file. 
https://gist.githubusercontent.com/gaearon/0275b1e1518599bbeafcde4722e79ed1/raw/db72dcbf3384ee1708c4a07d3be79860db04bff0/example.html

This workshop should focus on JSX and basic component design.
You can review the sheet to see what is covered in the lecture videos before this.

For this workshop, we should have campers build out a reusable navbar component

maybe something like this?

<img width="930" alt="Screenshot 2024-08-30 at 7 50 40 AM" src="https://github.com/user-attachments/assets/12ef9787-f8c3-44d2-9067-3624cac5a827">

It doesn't need to be super fancy.
The main goal is to get campers used to working with JSX which should be easier for them since it shares similar syntax to HTML

We should supply the CSS for them. And that will go inside the styles.css file in the editor like it normally does.

## Acceptance criteria

- [x] Add your project to the `frontend-cert/react-projects` folder

**Comment by jdwilkin4** [comment]
I am going to label all of these react and typescript workshops and labs as blocked for now until we know the final implementation in the editor by Oliver

**Comment by jdwilkin4** [comment]
Spoke with Oliver and have a better idea on how the editor is going to work for the react challenges. I have updated the description too.

So I am going to open this up and remove the Blocked label.

**Comment by jdwilkin4** [comment]
I am going to remove the blocked label because the react changes have been merged into main.
So we should created the workshop for this and see how everything works in the editor. If there are any issues, then we will need to reach out to Oliver. 

Once this workshop is merged into main, then we can open up the other workshops
